crafting-dead-core/src/main/java/com/craftingdead/core/client/ClientDist.java

```diff
@@ -568,7 +577,8 @@ public void handleRenderLiving(RenderLivingEvent.Pre<?, ?> event) {
     final var heldStack = event.getEntity().getMainHandItem();
     // TODO Unpleasant way of setting pose for gun. Introduce nicer system (with better poses).
     if (event.getRenderer().getModel() instanceof HumanoidModel<?> model
-        && heldStack.getItem() instanceof GunItem) {
+        && heldStack.getItem() instanceof GunItem
+        && !ENTITY_RENDER_BLACKLIST.contains(event.getEntity().getClass().getName())) {
```

Comment:
What's the purpose of this blacklist?

Reply:
The corpse mod uses the same model as the player, however when a gun is equipped, it looks odd a dead player aiming with a gun, the blacklist is just to prevent the corpse entity from having the aiming pose.